plugin.video.srf_ch_replay/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.srf_ch_replay" name="SRF Replay" version="2.0.0" provider-name="Man Behind MooN">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+	 <news>
+	</news>
+	<assets>
+		<icon>icon.png</icon>
+	</assets>
+    <summary lang="en_gb">SRF Replay plugin lists all shows from SRF.</summary>
+    <description lang="en_gb">SRF Replay plugin lists all recorded shows from the national Swiss German TV station SRF. All the shows are mainly in German.</description>
+    <disclaimer lang="en_gb">This plugin is not endorsed by SRF</disclaimer>
+	<summary lang="de_de">SRF Replay Plugin zeigt alle SRF Sendungen.</summary>
+    <description lang="de_de">Der SRF Replay Plugin zeigt alle aufgezeichneten Sendungen vom nationalen Schweizer deutschprachigen TV Sender SRF. Die Sendungen sind hauptsächlich in deutscher Spracher.</description>
+    <disclaimer lang="de_de">Dieser Plugin ist nicht offiziell von SRF</disclaimer>
+    <language>de_de</language>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
```suggestion
    <license>GPL-2.0-only</license>
```
or
```suggestion
    <license>GPL-2.0-or-later</license>
```
SPDX identifier

Reply:
changed to <license>GPL-2.0-or-later</license>